config/crd/bases/enterprise.splunk.com_licensemasters.yaml

```diff
@@ -1165,6 +1165,43 @@ spec:
                 - Always
                 - IfNotPresent
                 type: string
+              licenseManagerRef:
```

Comment:
This happens because `licenseM*Ref` is part of the `CommonSplunkSpec` so it comes out in all Splunk CRDs. This is the same behavior as all other Refs e.g. `clusterMasterRef` is part of the [Standalone CRD](https://github.com/splunk/splunk-operator/blob/93edb2ca16aba6bbab1ef3accd54532416db812f/config/crd/bases/enterprise.splunk.com_standalones.yaml#L966). There was no way around it without bigger changes. 

To avoid these cases we need to break out `CommonSplunkSpec` to fit only what's common to all Splunk CRDs. Maybe this type of refactoring can be done in the future.